CHANGELOG.md

```diff
@@ -1,9 +1,68 @@
 # nf-core/tools: Changelog
 
+## [v2.0 - Palladium Platypus](https://github.com/nf-core/tools/releases/tag/2.0) - [2021-07-09]
```

Comment:
Change the date here

Reply:
Maybe have a `Major Enhancements` section here given this is our first DSL2 release! e.g. [here](https://github.com/nf-core/viralrecon/blob/master/CHANGELOG.md#warning-major-enhancements)

Reply:
Yup good idea ... thought about it but then thought it might be enough in the release description. Not sure what people end up looking at? Can't hurt to have it in both though :)